app/commands/decidim/vocdoni/admin/save_results.rb

```diff
@@ -16,11 +16,8 @@ def initialize(form)
         #
         # Broadcasts :ok if setup, :invalid otherwise.
         def call
-          return broadcast(:invalid) if form.invalid?
-          return broadcast(:invalid) if form.results.empty?
-
           transaction do
-            save_results
+            SaveVocdoniElectionResultsJob.perform_later(election.id)
             change_election_status
```

Comment:
Can you move the `change_election_status` logic to `SaveVocdoniElectionResultsJob` 🙏🏽 ? I'm affraid that if we don't do it like that, we could have elections with results published but without actually the results (i.e. imagine if the job gave an error, if the Vocdoni API is down, or the Active Job worker is not running). 

We should change the flash alert too (with a message like "The results will be published in a few seconds") 

--- 

As this can be too much work we can move it out to an issue if you want.

Reply:
@microstudi I found this comment without answer. Does it make sense what I'm asking here?

Reply:
I'll resolve the conversation to not block this PR, but let me know if this should be moved to an issue if it may be a problem